x/evm/keeper/state_transition.go

```diff
@@ -415,6 +415,9 @@ func (k *Keeper) ApplyMessageWithConfig(ctx sdk.Context,
 		vmError = vmErr.Error()
 	}
 
+	// convert native events to ethereum logs
+	convertNativeEvents(stateDB, k.eventConverters)
```

Comment:
should we mv inside the commit

Reply:
not necessary I think

Reply:
Even if we don't commit in eth_call, we still want to see the logs